### ISSUE TITLE
Help. I don't have extra time to dedicate to this anymore.

### ISSUE THREAD

**Issue by pauldotknopf** [pauldotknopf/net-core-qml#240]
Let me say this. This project works great for my needs. I use it in production, and will likely use it for more projects going forward.

There are many people wanting features/things that are outside of scope of what **I need** for **my** projects.

After having a kid and playing live music more and more, I have less and less time that I'm willing to allocate to this project that doesn't serve my professional work.

With that said, I'm wondering if I should update the ```README.md``` to make this up-front and explicit. Thoughts?

**Comment by vadi2** [comment]
I think that would be a good idea :+1:

**Comment by timunie** [comment]
Hi @pauldotknopf 

If I were you I would write this on the main page. Moreover I would think about  using the discussion section of GitHub as some ideas may be useful for you as well. But this is just an opinion of a free time and self-taught programmer, so maybe it's different for you.

Anyway thank you for sharing this project.

Happy coding
Tim

**Comment by vadi2** [comment]
Could you think on how to do releases going forward? The un-released https://github.com/qmlnet/qmlnet/pull/219 adds simple window icon support, but not being on nuget, it's difficult to use.

**Comment by FieryRider** [comment]
Yeah, updating the `README.md` will be the best IMO. What about pull request? Will you merge those in the future?

**Comment by dev693** [comment]
Hi @pauldotknopf,
Maybe you remember we had a call and talked once about some bugfixes in Qml.Net because we use it as our UI Framework for our product Visoplan. I and our company will for sure spent time on this. So if you need Help I'm willing to help or even more.

**Comment by scherenhaenden** [comment]
I'm gonna fork this and see if I can come forward to update a few things and send a PR.

**Comment by MV10** [comment]
I'd have liked to see this disclosed in the README, too.

I have the time and programming experience to help out, but don't know anything about QT (yet), unfortunately.
